Feature/issue105

thuongho opened the pull request:
Update Footer #105

kengoy commented on components/Faq.vue:
```diff
@@ -8,11 +8,11 @@
-        :style="{ height: `${lastFaq.top + lastFaq.height * 2}px` }"
-      >
+    <div
+      :class="{ 'FaqContent-Scroll-Area': getScreenWidth() > 400 }"
```
Is there any reason of the 400 or 599 or 600 shows in the bottom code? I guess it is to distinguish mobile and desktop but I wonder why 3 types of width are shown in the same file.

thuongho replied:
@kengoy the 599 should have been 600, but it doesn't really change much. Basically, at 600px, the sidenavigation disappears. Once it disappears, I'm telling the code to not add more space for scrolling. The sidenavigation relies on the extra space to allow the last FAQ to scroll all the way to the top.

As for the 400px, I'm removing the scroll area for mobile. On mobile, it becomes a long single page instead of having a scrollable section like on desktop view for the FAQ page.

thuongho replied:
I changed the 599 to 600 to keep it consistent.

kengoy commented:
@thuongho  Thanks so much for your work. It looks good to my computer. Only one thing I want to fix is, on a high resolution screen, top page(index.vue) layout is unbalanced among the background bridge image and other components.  Attached patch will fix the issue if you could integrate it together.

![Screen Shot 2020-05-26 at 11 53 02 PM](https://user-images.githubusercontent.com/61864097/82987249-15f83200-9fac-11ea-8a31-dd1118e22b0d.png)

[index.vue.patch.txt](https://github.com/sfbrigade/stop-covid19-sfbayarea/files/4686981/index.vue.patch.txt)

thuongho commented:
@kengoy I'm looking into some edge cases. This PR is not ready.

thuongho commented:
@kengoy not sure why those files were added to my PR after I rebased with upstream/master yesterday. I just fetched upstream/master and rebased my branch again.
